docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
       - 8080:80
       - 8443:443
     restart: always
+    volumes:
+      - "./shared/homeless/web/uploads:/var/www/symfony/web/uploads"
```

Comment:
can we use simpler input directory? Like `./homeless/uploads` instead of `./shared/homeless/web/uploads`